src/dev-app/index.html

```diff
@@ -17,7 +17,7 @@
       window.customElements.forcePolyfill = true;
     }
   </script>
-  <script src="@webcomponents/custom-elements/custom-elements.min.js"></script>
+<!--  <script src="@webcomponents/custom-elements/custom-elements.min.js"></script>-->
```

Comment:
Was this causing a conflict?

Reply:
This change wasn't intended to be uploaded. The line was commented out during a performance audit. I noticed the `custom-elements` script was running code during change detection while profiling the virtual scroll table.

I'll revert this change.